app/controllers/application_controller.rb

```diff
@@ -12,4 +14,8 @@ def http_basic_authenticate
         password == ENV["HTTPAUTH_PASSWORD"].to_s
     end
   end
+
+  def handle_session_expired
+    redirect_to root_path
```

Comment:
Can be one for later, but I think it would be worth monitoring how often this happens, either with a log statement or by exposing a metric to prometheus

Reply:
have updated with error logging and redirects to session-expiry page, with a link to restart to home page